src/Hangfire.Tags.PostgreSql/GlobalConfigurationExtensions.cs

```diff
@@ -21,6 +21,7 @@ public static IGlobalConfiguration UseTagsWithPostgreSql(this IGlobalConfigurati
             sqlOptions = sqlOptions ?? new PostgreSqlStorageOptions();
 
             options.Storage = new PostgreSqlTagsServiceStorage(sqlOptions);
+            options.RetryPolicy = options.RetryPolicy ?? new TagsStorageTransactionRetryPolicy(15);
```

Comment:
I'm not sure about the default retry count. `15` seems to work well in the unit test. But maybe we want to have a higher value, just in case.